RATreeView/RATreeView/RATreeView.h

```diff
@@ -413,7 +411,6 @@ typedef enum RATreeViewRowAnimation {
  *  Asks the delegate if the row for a specified item should be highlighted.
  *
  *  @param treeView     The tree-view object that is making this request.
- *  @param treeNodeInfo Object including additional information about item.
  *
```

Comment:
Include this also:
*  @param item         An item identifying a cell in tree view.